server/tables/tables.go

```diff
@@ -305,7 +305,8 @@ type APIKey struct {
 	//
 	// NOTE: If the default is changed, a DB migration may be required to
 	// migrate old DB rows to reflect the new default.
-	Capabilities int32 `gorm:"default:1"`
+	Capabilities        int32 `gorm:"default:1"`
+	VisibleToDevelopers bool
```

Comment:
This field will be NULL by default -- dealing with nulls can sometimes be a pain, so IMO it'd be better to add the annotation `gorm:"not null"` and GORM will make this value `false` for existing rows instead

Also just double-checking, do you want this to be set to false for existing rows?

Reply:
Yeah, I definitely want all API keys that are visible to developers to be manually marked as such.

I've added `gorm:"not null;default:false"` to make the default value false.